manticore/core/smtlib/constraints.py

```diff
@@ -116,7 +116,7 @@ def _get_sid(self) -> int:
         self._sid += 1
         return self._sid
 
-    def related_to(self, related_to: Optional[Expression] = None):
+    def related_to(self, related_to: Optional[Expression] = None) -> ConstraintSet:
```

Comment:
My bad. Forgot this was inside the class
```suggestion
    def related_to(self, related_to: Optional[Expression] = None) -> "ConstraintSet":
```